frontend/src/components/pages/logHours.tsx

```diff
@@ -72,6 +80,63 @@ export default function LogHours({ eventData }: LogHoursProps) {
     return event._id === eventId;
   });
 
+  const addToUser = async (id: string) => {
+    await fetch(`http://localhost:3001/users/${currentUser._id}`, {
+      method: 'PUT',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ shiftId: id }),
+    });
+  };
+
+  const updateUserHours = async () => {
+    await fetch(`http://localhost:3001/users/${currentUser._id}/test`, {
+      method: 'PUT',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ numHours: hours }),
+    });
+  };
+
+  const addToEvent = async (id: string) => {
+    await fetch(`http://localhost:3001/events/${eventId}`, {
+      method: 'PUT',
+      headers: {
+        'Content-Type': 'application/json',
```

Comment:
These can be combined into one endpoint, just pass all the data in the  request body from the frontend.